openedx/core/djangoapps/credit/views.py

```diff
@@ -29,6 +29,59 @@
 log = logging.getLogger(__name__)
 
 
+@require_GET
+def get_provider_detail(request, provider_id):
```

Comment:
Is there a particular reason Django REST Framework is not being used here or elsewhere for these views?